### PR TITLE
Explicit CreateCluster & UpdateCluster functions

### PR DESCRIPTION
**Pull request by justinsb**
This enables better validation (in the etcd status PR)

### CODE REVIEW

File: pkg/client/simple/clientset.go [hunk 1]
```diff
@@ -57,16 +60,21 @@ type RESTClientset struct {
...
-	namespace := restNamespaceForClusterName(cluster.Name)
-	return c.KopsClient.Clusters(namespace)
-}
-
 func (c *RESTClientset) GetCluster(name string) (*kops.Cluster, error) {
 	namespace := restNamespaceForClusterName(name)
 	return c.KopsClient.Clusters(namespace).Get(name, metav1.GetOptions{})
 }
 
+func (c *RESTClientset) CreateCluster(cluster *kops.Cluster) (*kops.Cluster, error) {
```
Comment (chrislovecnm) [inline pkg/client/simple/clientset.go]: You want to do a follow-up PR on the code comments?

Comment (justinsb) [inline pkg/client/simple/clientset.go]: I'll add them.  I wish go tooling understood interfaces ;-)

**Comment by justinsb** [comment]
Added godoc comments; PTAL @chrislovecnm

**Comment by chrislovecnm** [comment]
/retest

Blah flakes... we are now running the dev shas of k8s, for exampe `v1.8.0-alpha.3.116+657db0eae7995a`, during e2e run and not the latest tag.  I think we should go back to running the latest tag ...

**Pull request merged**